docs/certbot.rst

```diff
@@ -0,0 +1,52 @@
+Certbot options
+```````````````
+
+The certbot playbook
+~~~~~~~~~~~~~~~~~~~~
+
+As a convenience, the Plone Ansible Playbook kit includes a separate
+playbook that will install certbot-nginx and create certificates as necessary for specified hostnames.
+
+The certbot playbook currently only supports Debian-family target servers.
+
+To use the certbot playbook, edit your ``local-configure.yml`` file to add a ``certbot_hosts`` list variable containing an entry for each hostname for which you wish to get a certbot certificate:
+
+.. code-block:: yaml
+
+   certbot_hosts:
+     - one.mcsmith.org
+     - two.mcsmith.org
+
+Run the playbook as you would the main playbook, adding whatever command-line switches you need (like ``-k`` or ``-K``):
+
+.. code-block:: console
+
+    ansible-playbook -k certbot.yml
+
+This will first install ``python3-certbot-nginx`` from the certbot/certbot ppa.
+Then it will create certificates as necessary for each hostname in the ``certbot_hosts`` list.
+If a certificate already exists, it will not attempt addition.
+
+Note that ``python3-certbot-nginx`` includes an auto-renewal cronjob.
```

Comment:
I did not see this on my server in `/var/spool/cron/crontabs/` after running the playbook. Where is it located?

Reply:
/etc/cron.d/certbot

Reply:
Got it.

```bash
SHELL=/bin/sh
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin

0 */12 * * * root test -x /usr/bin/certbot -a \! -d /run/systemd/system && perl -e 'sleep int(rand(43200))' && certbot -q renew
```

I note that the cron job does not include `--nginx` flag for `certbot -q renew`. I guess we will find out if it works for me on May 12 when my current cert is due to expire.

Reply:
The auto-renew cronjob failed. See https://github.com/plone/ansible-playbook/pull/136#issuecomment-627234563